blocks/library/button/editor.scss

```diff
@@ -15,7 +15,7 @@ $blocks-button__height: 46px;
 		z-index: 2;
 		top: $blocks-button__height + 2px;
 		left: 50%;
-		transform: translateX( -50% );
```

Comment:
Which bug is this fixing exactly? and why `13%`?

Reply:
This fix for #2635